setup.cfg

```diff
@@ -33,7 +33,7 @@ test = pytest
 markers:
   skip_if_np_ge_114: Skip a test when NumPy is older than 1.14
   skip_if_np_lt_114: Skip a test when NumPy is at least 1.14
-addopts = -rsx -v --durations=10
+addopts = -rsx --durations=10
 # minversion = 3.2
```

Comment:
Just wondering why this commented code is here. Perhaps it's related to #4142?

Reply:
The filterwarnings bit was done to help elevate warnings to errors to remove them from the test suite (See #4142 )

I suspect that the `minversion` note is to ensure that a minimum version of pytest is used when doing this, but honestly I'm not sure.  I think that this was copied over from dask-ml.